services-turf/src/main/java/com/mapbox/turf/TurfMeta.java

```diff
@@ -142,6 +145,81 @@ private TurfMeta() {
     return coords;
   }
 
+  /**
+   * Get all coordinates from a {@link Feature} object, returning a {@link List} of {@link Point}
+   * objects.
+   *
+   * @param feature the {@link Feature} that you'd like to extract the Points from.
+   * @return a {@code List} made up of {@link Point}s
+   * @since 4.7.0
+   */
+  public static List<Point> coordAll(@NonNull Feature feature) {
+    return addCoordAll(new ArrayList<Point>(), feature);
+  }
+
+  /**
+   * Get all coordinates from a {@link FeatureCollection} object, returning a
+   * {@link List} of {@link Point} objects.
+   *
+   * @param featureCollection the {@link FeatureCollection} that you'd like
+   *                          to extract the Points from.
+   * @return a {@code List} made up of {@link Point}s
+   * @since 4.7.0
+   */
+  public static List<Point> coordAll(@NonNull FeatureCollection featureCollection) {
+    List<Point> finalCoordsList = new ArrayList<>();
+    for (Feature singleFeature : featureCollection.features()) {
+      finalCoordsList.addAll(addCoordAll(finalCoordsList, singleFeature));
+    }
+    return finalCoordsList;
+  }
+
+  private static List<Point> addCoordAll(@NonNull List<Point> pointList, @NonNull Feature feature) {
+    pointList.addAll(coordAllFromSingleGeometry(feature.geometry()));
+    return pointList;
+  }
+
+  /**
+   * Get all coordinates from a {@link Geometry} object, returning a {@link List} of {@link Point}
+   * objects.
+   *
+   * @param geometry the {@link Geometry} object to extract the {@link Point}s from
+   * @return a {@code List} made up of {@link Point}s
+   * @since 4.7.0
+   */
+  private static List<Point> coordAllFromSingleGeometry(@NonNull Geometry geometry) {
```

Comment:
Add `List<Point> to the list of parameters, then it will look something like:
```
  private static List<Point> coordAll(@NonNull List<Point> pointList, @NonNull Geometry geometry) {
    if (geometry instanceof Point) {
      pointList.add((Point) geometry);

    } else if (geometry instanceof MultiPoint) {
      pointList.addAll(((MultiPoint)geometry).coordinates());

    } else if (geometry instanceof LineString) {
      pointList.addAll(((LineString)geometry).coordinates());

    } else if (geometry instanceof MultiLineString) {
      coordAll(pointList, (MultiLineString) geometry);

    } else if (geometry instanceof Polygon) {
      coordAll(pointList, (Polygon) geometry);

    } else if (geometry instanceof MultiPolygon) {
      coordAll(pointList, (MultiPolygon) geometry);

    } else if (geometry instanceof GeometryCollection) {
      // recursive
      for (Geometry singleGeometry : ((GeometryCollection) geometry).geometries()) {
        coordAll(pointList, singleGeometry);
      }
    }
    return pointList;
  }
```